docs/low-code-sql/data-tests/data-tests.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Data tests
+id: data-tests
+description: Data tests check the validity of your SQL
+sidebar_position: 8
+tags:
+  - concept
+  - testing
+  - sql
+---
+
+You can use data tests to ensure that your business data is generated reliably over time. As a data engineer, data analyst, or business user, you can run data tests so that you don’t have to manually check every Dataset every time you run a Job or model. The data test checks the validity of the SQL in your project.
+
+A data test is an assertion you make about a Dataset in your project. The Dataset can be the output from a series of transformations, or the Dataset can be a particular data source, seed, or model.
+
+For example, find the following Test named `ref_int_orders_customers` which checks the validity of the SQL in the `HelloWorld_SQL` Project. In particular, the referential integrity check for orders and customers asserts that every `customer_id` entry in the `orders` table is present in the `customers` table.
```

Comment:
"find the following Test"
I want to make sure users don't try to find this test in the HelloWorld_SQL.

Reply:
Done.